runbook.md

```diff
@@ -0,0 +1,105 @@
+# next n-ui
+
+Server, build and client side bootstrapping for ft.com’s user-facing applications.
+
+## Primary URL
+
+https://www.npmjs.com/package/@financial-times/n-ui
+
+## Service Tier
+
+Bronze
+
+## Lifecycle Stage
+
+Production
+
+## Delivered By
+
+next
+
+## Supported By
+
+next
+
+## Known About By
+
+- rhys.evans
+- dora.militaru
+- matt.hinchliffe
+- maggie.allen
+
+## Contains Personal Data
+
+false
+
+## Contains Sensitive Data
+
+false
+
+## Host Platform
+
+npm
+
+## Architecture
+
+## First Line Troubleshooting
+Problems with n-ui are likely to manifest in the user-facing applications which depend on it. Please follow the troubleshooting guide for the affected application.
+
+## Second Line Troubleshooting
+
+### General tips for troubleshooting Customer Products systems
+
+- [Out of hours runbook for FT.com (wiki)](https://customer-products.in.ft.com/wiki/Out-of-hours-Runbook)
+- [General tips for debugging FT.com (wiki)](https://customer-products.in.ft.com/wiki/Debugging-Tips)
+- [General information about monitoring and troubleshooting FT.com systems (wiki)](https://customer-products.in.ft.com/wiki/Monitoring-and-Troubleshooting-systems)
+
+
+## Monitoring
+
+None
+
+## Data Recovery Process Type
+
+Not applicable
+
+## Release Process Type
+
+PartiallyAutomated
+
+## Rollback Process Type
+
+PartiallyAutomated
+
+## Release Details
+
+This app is released using Circle CI.
+Rollback is done manually on Github. See [the guide on the wiki](https://customer-products.in.ft.com/wiki/How-does-deploying-our-Heroku-apps-work%3F) for instructions on how to deploy.
+
+When you release an n-ui tag, 3 things happen:
+
+- assets are built and deployed to s3, from where they are linked to/downloaded by apps
+- the npm package is published
+- during work hours (9am to 4pm), all user-facing apps are rebuilt to pick up the changes
+
+https://github.com/Financial-Times/n-ui#releasing-n-ui
+
+
+## Key Management Process Type
+
+PartiallyAutomated
+
+## Key Management Details
+
+You can read about how to rotate an AWS key [over on the Customer Products Wiki](https://customer-products.in.ft.com/wiki/Rotating-AWS-Keys)
+See the Customer Products [key management and troubleshooting wiki page](https://customer-products.in.ft.com/wiki/Key-Management-and-Troubleshooting)
+
+## More Information
+
+n-ui release failure causes n-ui parity issues for anyone trying to build a user-facing app.
+
+There are no monitoring services set up for n-ui. Services which depend on it are individually monitored.
+
+Assets generated by n-ui are stored in a [hashed assets](https://biz-ops.in.ft.com/System/next-hashedasset) bucket which is monitored as Platinum service.
```

Comment:
This should also be declared as a relationship with the next-hashedasset system under the `Dependencies` section.